### PR TITLE
dev-libs/wlc: version bump to 0.0.5

### PR DESCRIPTION
**Pull request by Hummer12007**

### CODE REVIEW

File: dev-libs/wlc/wlc-0.0.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		x11-libs/libxkbcommon
+		x11-misc/xkeyboard-config
+		dev-libs/libinput
+		dev-libs/wayland
+		X? ( x11-libs/libX11
+			 x11-libs/libxcb
+			 x11-libs/xcb-util-image
+			 x11-libs/xcb-util-wm
+			 x11-libs/libXfixes )
+		systemd? ( sys-apps/systemd sys-apps/dbus )"
```
Comment (SoapGentoo) [inline dev-libs/wlc/wlc-0.0.5.ebuild]: remove one tab for the whole `RDEPEND` block

**Comment by Whissi** [comment]
@tajjada You are proxy-maintaining this package. Do you agree with this bump?

**Comment by Hummer12007** [comment]
@kensington @monsieurp can you finally merge? The proxied-maintainer is AWOL.

**Comment by tajjada** [comment]
@Hummer12007 @kensington @monsieurp 

I was away for a couple of weeks and did not have internet access for most of it. I am back now. I am sorry for this, I probably should have informed people before leaving.

Please merge this.

**Comment by monsieurp** [comment]
:+1:

**Pull request closed without merging**